cli/dcoscli/task/main.py

```diff
@@ -1,26 +1,36 @@
-"""Get the status of DCOS tasks
+"""Manage DCOS tasks
 
 Usage:
     dcos task --info
     dcos task [--completed --json <task>]
+    dcos task log [--completed --follow --lines=N] <task> [<file>]
 
 Options:
     -h, --help    Show this screen
     --info        Show a short description of this subcommand
+    --completed   Include completed tasks as well
```

Comment:
There are no "inactive" tasks

And no, we shouldn't.  Users might just be interested in active (running) tasks.